lib/itamae/resource/file.rb

```diff
@@ -166,7 +166,7 @@ def send_tempfile
           src = if content_file
                   content_file
                 else
-                  f = Tempfile.open('itamae')
+                  f = Tempfile.open('itamae', :mode=>IO::RDWR | IO::CREAT | IO::TRUNC | IO::BINARY)
```

Comment:
1. I think this behavior should be optional, not by default.
2. Is not specifying only `IO::BINARY` sufficient for this purpose?